Extensions/combined/_locales/fr/messages.json

```
@@ -0,0 +1,35 @@
+{
+    "extensionName": {
+        "message": "Retour YouTube Dislike"
+    },
+    "extensionDesc": {
+        "message": "Rend la capacité de voir les aversions"
```

```suggestion
        "message": "Ré-affiche les pouces rouges/dislikes des vidées"
```
Note: "pouce rouge" == "red thumb" is the common name for dislikes in the french youtube community, tha's why I think it should be used here.